implementation.js

```diff
@@ -19,11 +19,21 @@ var slice = Function.call.bind(Array.prototype.slice);
 var concat = Function.call.bind(Array.prototype.concat);
 var forEach = Function.call.bind(Array.prototype.forEach);
 
-var hasSymbols = require('has-symbols')();
+var hasSymbols = typeof global.Symbol !== 'function';
```

Comment:
this will cause it to pass on a symbol sham; the intention is to only have it be true for native symbols. (also, you inverted the logic, breaking it)
```suggestion
var hasSymbols = typeof Symbol === 'function' && typeof Symbol() === 'symbol';
```